app/services/deploy_service.rb

```diff
@@ -119,4 +120,9 @@ def choose_assignee(pull_request)
       .group_by(&:login).map { |k, v| [v.size, k] }.sort.reverse.map(&:last)
       .detect { |l| github_client.check_assignee(github_repo, l) }
   end
+
+  def can_release_now?(buckets, reference_time)
+    time_wa = reference_time.beginning_of_minute.strftime('%a, %d %b %Y %H:%M')
```

Comment:
Why convert to a string here?

Bonus: you can provide a default argument of `reference_time = Time.now` so that callers don't have to pass the 2nd argument (unless they choose to).

Reply:
just added the default, I used to have defaults on python...completely forgot that ruby also supports it.